source/Halibut.Tests/Support/BackwardsCompatibility/PreviousVersions.cs

```diff
@@ -1,11 +1,69 @@
-﻿namespace Halibut.Tests.Support.BackwardsCompatibility
+﻿using System;
+
+namespace Halibut.Tests.Support.BackwardsCompatibility
 {
     public static class PreviousVersions
     {
+        // The earliest release with a stable WebSocket implementation
+        static readonly Version v6_0_658_WebSocket_Stability_Fixes = new("6.0.658");
+
         // Version prior to new exception types being added to Halibut. All exceptions are HalibutClientException
-        public const string v5_0_429 = "5.0.429";
+        public static readonly HalibutVersions v5_0_429 = new (new Version("5.0.429"), pollingOverWebSocketServiceVersion: v6_0_658_WebSocket_Stability_Fixes);
```

Comment:
I prefer this way as it's really clear to see what version you will end up testing from the PreviousVersions